tooling/minimize/src/rvalue.rs

```diff
@@ -38,12 +38,12 @@ impl<'cx, 'tcx> FnCtxt<'cx, 'tcx> {
                     (ShlUnchecked, Type::Int(_)) => BinOp::Int(IntBinOp::ShlUnchecked),
                     (ShrUnchecked, Type::Int(_)) => BinOp::Int(IntBinOp::ShrUnchecked),
 
-                    (Lt, Type::Int(_)) => BinOp::IntRel(IntRel::Lt),
-                    (Le, Type::Int(_)) => BinOp::IntRel(IntRel::Le),
-                    (Gt, Type::Int(_)) => BinOp::IntRel(IntRel::Gt),
-                    (Ge, Type::Int(_)) => BinOp::IntRel(IntRel::Ge),
-                    (Eq, Type::Int(_)) => BinOp::IntRel(IntRel::Eq),
-                    (Ne, Type::Int(_)) => BinOp::IntRel(IntRel::Ne),
+                    (Lt, Type::Int(_)) | (Lt, Type::Bool) => BinOp::IntRel(IntRel::Lt),
```

Comment:
```suggestion
                    // We use the int relations als for bools
                    // (and insert a cast, see below).
                    (Lt, Type::Int(_) | Type::Bool) => BinOp::IntRel(IntRel::Lt),
```

Reply:
Note that I didn't just add a comment, I also suggested a way to simplify the pattern. ;)

Reply:
ah nice :)